config/locales/views/home/home.yml

```diff
@@ -4,10 +4,10 @@ en:
     download: Download
     feature:
       description_html: |-
-        How can schools cut energy costs and reduce emissions? How can teachers inspire their pupils to become the sustainability leaders of tomorrow?
+        Read our case studies for examples of how schools and Multi-Academy Trusts have used Energy Sparks to achieve amazing energy and carbon savings.
```

Comment:
I am thinking 'Multi-Academy Trusts' ought not to be capitalised here, just waiting for Claudia to reply on this